tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -267,6 +268,16 @@ func NewBalanceLeaderSchedulerCommand() *cobra.Command {
 	return c
 }
 
+// NewEvictSlowStoreSchedulerCommand returns a command to add a evict-slow-store-scheduler.
+func NewEvictSlowStoreSchedulerCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "evict-slow-store-scheduler",
```

Comment:
the schedule should add store_id param???

Reply:
No, it just needs the id of the previous evicted store which was stored in etcd.